File: Week 1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = 0; i < height; i++)
    {


        for (int j = height; j > 0; j--)
        {
            if (j <= i + 1)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: Week 6/sentimental-mario-more/mario.py
from cs50 import get_int
while True:
    x = get_int("Height: ")
    if x > 0 and x < 9:
        break

for i in range(1, x + 1):
    print((" " * (x - i)) + ("#" * i) + "  " + ("#" * i))
